eth/stagedsync/exec3.go

```diff
@@ -306,7 +306,7 @@ func ExecV3(ctx context.Context,
 	var count uint64
 	var lock sync.RWMutex
 
-	shouldReportToTxPool := maxBlockNum-blockNum <= 8
+	shouldReportToTxPool := maxBlockNum-blockNum <= 64
```

Comment:
lets put this into a constant? I think we can gather all those constants later on into one place to ease configuration. Could be made as a follow up as well.